SIT/smoke_inflight/06_vector/cdm.migrateData.assert

```diff
@@ -0,0 +1,4 @@
+Read Record Count: 2
```

Comment:
These whole files are being added to `smoke_inflight` which isn't being tested actively, but will be moved to `smoke` directory once we've a docker container that supports vector cql data type.